packages/core/content-manager/server/src/policies/hasPermissions.ts

```diff
@@ -7,6 +7,10 @@ const { createPolicy } = policy;
 export default createPolicy({
   name: 'plugin::content-manager.hasPermissions',
   validator: validateHasPermissionsInput,
+  /**
+   * NOTE: Action aliases are currently not checked at this level (policy).
+   *       If you want to add support for it, please create a dedicated RFC with a list of potential side effect this could have.
+   */
```

Comment:
Should we say that it is actually intended? That is, because aliases are internal only, they will not be visible to policies.